Rakefile

```diff
@@ -14,18 +14,7 @@ unless Rails.env.production?
   require 'rspec/core/rake_task'
   task(:spec).clear
   RSpec::Core::RakeTask.new(:spec) do |t|
-    t.pattern = Dir.glob(
-      [
-        'spec/**/*_spec.rb',
-        'modules/vba_documents/spec/**/*_spec.rb',
-        'modules/appeals_api/spec/**/*_spec.rb',
-        'modules/claims_api/spec/**/*_spec.rb',
-        'modules/va_facilities/spec/**/*_spec.rb',
-        'modules/veteran/spec/**/*_spec.rb',
-        'modules/veteran_verification/spec/**/*_spec.rb',
-        'modules/openid_auth/spec/**/*_spec.rb'
-      ]
-    )
+    t.pattern = Dir.glob(['spec/**/*_spec.rb', 'modules/*/spec/**/*_spec.rb'])
```

Comment:
Nice! Its so much easier to support convention (specs in modules get run by RSpec) vs configuration ('remember to add your module specs to the Rakefile')

👍